### ISSUE TITLE
Question relating to Implementation of MaskL1Loss used in DBLoss

### ISSUE THREAD

**Issue by xaclincoln** [PaddlePaddle/PaddleOCR#5898]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem
Sum is used in forward method of MaskL1Loss class without any axis specified , as in the following:

`

class MaskL1Loss(nn.Layer):
    def __init__(self, eps=1e-6):
        super(MaskL1Loss, self).__init__()
        self.eps = eps

    def forward(self, pred, gt, mask):
        """
        Mask L1 Loss
        """
        loss = (paddle.abs(pred - gt) * mask).sum() / (mask.sum() + self.eps) //
        loss = paddle.mean(loss)
        return loss

`

As the [paddle doc tells](https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/sum_en.html),  if axis is None, sum all elements of x and return a Tensor with a single element, otherwise must be in the range [−rank(x),rank(x)).
Sum is followed by mean function , but loss returned from previous operation is a scalar. In my view, it's meaningless to invoke mean for a scalar, shouldn't we specify axis=0 to calculate loss of every single sample , like the following?

`

class MaskL1Loss(nn.Layer):
    def __init__(self, eps=1e-6):
        super(MaskL1Loss, self).__init__()
        self.eps = eps

    def forward(self, pred, gt, mask):
        """
        Mask L1 Loss
        """
        loss = (paddle.abs(pred - gt) * mask).sum(0) / (mask.sum(0) + self.eps) //here, axis is specified
        loss = paddle.mean(loss) //here, mean loss of all samples in a batch
        return loss

`

- 系统环境/System Environment：windows 10
- 版本号/Version：Paddle：2.2.1  PaddleOCR：2.4 问题相关组件/Related components：Loss used in DB Text Detection
- 运行指令/Command Code：
- 完整报错/Complete Error Message：
